buildutils/src/utils.ts

```diff
@@ -15,10 +15,13 @@ export function getLernaPaths(basePath = '.'): string[] {
   basePath = path.resolve(basePath);
   let baseConfig = require(path.join(basePath, 'package.json'));
   let paths: string[] = [];
-  let packages =
-    baseConfig.workspaces.packages ||
-    baseConfig.workspaces ||
-    baseConfig.packages;
+  let packages;
+  if (baseConfig.workspaces) {
+    packages = baseConfig.workspaces.packages || baseConfig.workspaces;
+  } else {
+    let baseConfig = require(path.join(basePath, 'lerna.json'));
```

Comment:
Is this line redundant?

Reply:
`lerna.json` is the significant part here (that actually fixes the issue).

Reply:
Possibly we would want a try/except to catch the file not existing, before giving a helpful error message.